courses/machine_learning/deepdive2/image_classification/labs/1_mnist_linear.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<p>Should be TODO 1? Seems to match with learning objective at top as well as solution.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>